tesla_ble_mqtt/config.yaml

```diff
@@ -17,33 +17,30 @@ map:
   - share:rw
 startup: services
 options:
-  vin1: ""
-  vin2: ""
-  vin3: ""
-  presence_detection: false
-  ble_mac1: ""
-  ble_mac2: ""
-  ble_mac3: ""
+  vin: ""
+  vin_list: ""
+  ble_mac: ""
+  ble_mac_list: ""
+  presence_detection_ttl: 180
   debug: false
   mqtt_ip: ""
   mqtt_port: "1883"
   mqtt_user: ""
   mqtt_pwd: ""
   send_cmd_retry_delay: "5"
+
 schema:
-  vin1: str?
-  vin2: str?
-  vin3: str?
-  presence_detection: bool
-  ble_mac1: str?
-  ble_mac2: str?
-  ble_mac3: str?
+  vin: str?
+  vin_list: "match(^([A-HJ-NPR-Z0-9]{17})(\|[A-HJ-NPR-Z0-9]{17})*$)"
+  ble_mac:str?
+  ble_mac_list: "match(^([0-9A-Fa-f]{2}(:[0-9A-Fa-f]{2}){5})(\|[0-9A-Fa-f]{2}(:[0-9A-Fa-f]{2}){5})*$)?"
```

Comment:
Need to be tested

Reply:
Not sure how to believe here ... ChatGPT gave me a Python script to make the test.
```
Generic: (^([0-9A-Fa-f]{2}(:[0-9A-Fa-f]{2}){5})(\|[0-9A-Fa-f]{2}(:[0-9A-Fa-f]{2}){5})*$)
Python : (^([0-9A-Fa-f]{2}[:-]){5}([0-9A-Fa-f]{2})(\|([0-9A-Fa-f]{2}[:-]){5}([0-9A-Fa-f]{2}))*$
```

Testing w/ python script both works.

```
import re

# Generic
mac_pattern = re.compile(r'^([0-9A-Fa-f]{2}[:-]){5}([0-9A-Fa-f]{2})(\|([0-9A-Fa-f]{2}[:-]){5}([0-9A-Fa-f]{2}))*$')
mac_pattern = re.compile(r'(^([0-9A-Fa-f]{2}(:[0-9A-Fa-f]{2}){5})(\|[0-9A-Fa-f]{2}(:[0-9A-Fa-f]{2}){5})*$)')

# Test MAC Addresses
mac_test_1 = "00:1A:2B:3C:4D:5E"
mac_test_2 = "00:1A:2B:3C:4D:5E|00:1A:2B:3C:4D:5F"
```

Some manual work involved above, I added a | in beginning then at end , other changes and it seems to work as well.

Reply:
Should we keep this variable?

Reply:
Yes for now. We still need MAC until next re-work of the ble presence loop that I'll do ... this weekend or ?!